db/migrate/20220212185723_create_users.rb

```diff
@@ -0,0 +1,8 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] The users table must have the following fields: `name`, `photo`,  `bio`, and `post_counter`. Kindly update the migration to accommodate these fields as per the requirements.